checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -56,3 +56,87 @@ class Command {
         return [Command]::new($_executable, $_argumentsList)
     }
 }
+
+function StartSchedulerRunner {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true)]
+        [string]$configFilePath
+    )
+
+    $configFileContent = [Config]::ParseConfigFile($configFilePath)
```

Comment:
`config = ...`
We get an actual `Config` object here, right?

Reply:
I don't know what exactly you are reffering to.

I created it so that we just need the path to the config file as an input to start the scheduler-runner, so we expect a string. Then we can just start a process with `StartSchedulerRunner($pathToConfig)`.

Reply:
I was referring to the variable name `$configFileContent`. `[Config]::ParseConfigFile($configFilePath)` returns a `Config` object, right? So `$config` seems to be a more fitting name. `$configFileContent` would suggest that we get the raw, unparsed file content.